python/hail/dataset.py

```diff
@@ -5214,6 +5214,217 @@ def vep(self, config, block_size=1000, root='va.vep', csq=False):
         return VariantDataset(self.hc, jvds)
 
     @handle_py4j
+    def nirvana(self, config, block_size = 50000, root = 'va.nirvana'):
+        """Annotate variants with `Nirvana <https://github.com/Illumina/Nirvana>`_.
+
+        ***Configuration***
```

Comment:
Here we can link to the documentation that Nirvana prepares, which should more fully explain how to set up the dependencies.